cmd/generate.go

```diff
@@ -103,6 +104,49 @@ func genProcessCluster(cmd *cobra.Command, clusterName string, p *ants.Pool) {
 	}
 	sort.Strings(compList) // process components in sorted order
 
+	// List of cluster components in kr8
+	var clComps []string
+	for c, _ := range clusterComponents {
+		clComps = append(clComps, c)
```

Comment:
Why create clComps when you have clusterComponents?

Reply:
couldn't use clusterComponents directly on loop below as it was with var type gjson.Result. So created a new slice

Reply:
clComp looks like `compList` from https://github.com/apptio/kr8/pull/40/files#diff-d2b0f452bba73e3d2e556bae31fdf25c9c5375685adf1617bf27ceed1c66d031R102

Reply:
done